src/Components/CustomTable.tsx

```diff
@@ -53,14 +70,30 @@ const CustomTable: React.FC<TableProps> = ({ headers }) => {
                         </TableRowStyle>
                     </TableHead>
                     <TableBody>
-                        {filterData.slice(page * 5, page * 5 + 5).map((Data) => (
-                            <TableRow key={Data.id}>
-                                {Object.keys(Data).map((key) => (
-                                    <TableCell key={key}>{Data[key]}</TableCell>
-                                ))}
-                                <ButtonStyle onClick={() => navigate(generatePath(APP_ROUTES.DETAILS_PAGE,{id:Data.id}))}>Description</ButtonStyle>
-                            </TableRow>
-                        ))}
+                        {filterData
```

Comment:
this component is only suppose to map data please handle data changing logic from parent